Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1921,6 +1921,14 @@
     <string name="duration_dialog_ok_button">OK</string>
     <!-- NOTR -->
     <string name="duration_dialog_default_value">00:00</string>
+    <!-- XTXT: Duration hours text -->
+    <string name="duration_hours">"Stunden"</string>
+    <!-- XTXT: Duration minutes text -->
+    <string name="duration_minutes">"Minuten"</string>
+    <!-- XTXT: Duration Organize title -->
+    <string name="duration_typical_lenght_of_stay">"Typische Aufenthaltsdauer"</string>
```

Comment:
These last two Strings don't belong in this PR I guess.

Reply:
yeah, they don't need to be here